src/components/Navbar.tsx

```diff
@@ -0,0 +1,40 @@
+import classNames from 'classnames';
+import { FC } from 'react';
+import { NavLink } from 'react-router-dom';
+
+export const Navbar: FC = () => {
+  return (
+    <nav
+      data-cy="nav"
+      className="navbar is-fixed-top has-shadow"
+      role="navigation"
+      aria-label="main navigation"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <NavLink
+            className={({ isActive }) => classNames(
+              'navbar-item', {
+                'has-background-grey-lighter': isActive,
+              },
+            )}
+            to="/"
+          >
+            Home
+          </NavLink>
+
+          <NavLink
+            className={({ isActive }) => classNames(
+              'navbar-item', {
+                'has-background-grey-lighter': isActive,
+              },
+            )}
+            to="/people"
+          >
+            People
+          </NavLink>
```

Comment:
A good idea will be to create a custom component for this to avoid code duplication. Rewatch [this video](https://mate.academy/learn/react/react-router?section=video&videoId=3348)